Sources/Apodini/Semantic Model Builder/GraphQL/GraphQLInterfaceExporter.swift

```diff
@@ -0,0 +1,150 @@
+    }
+
+    private func graphQLIDE(_ _: Vapor.Request) -> Vapor.Response {
+        let html: Vapor.Response.Body = """
+                                        <html>
+                                          <head>
+                                            <title>GraphiQL</title>
+                                            <link href="https://unpkg.com/graphiql/graphiql.min.css" rel="stylesheet" />
+                                          </head>
+                                          <body style="margin: 0;">
+                                            <div id="graphiql" style="height: 100vh;"></div>
+                                            <script
+                                              crossorigin
+                                              src="https://unpkg.com/react/umd/react.production.min.js"
+                                            ></script>
+                                            <script
+                                              crossorigin
+                                              src="https://unpkg.com/react-dom/umd/react-dom.production.min.js"
+                                            ></script>
+                                            <script
+                                              crossorigin
+                                              src="https://unpkg.com/graphiql/graphiql.min.js"
+                                            ></script>
+                                            <script>
+                                              const graphQLFetcher = graphQLParams =>
+                                                fetch('/graphql', {
+                                                  method: 'post',
+                                                  headers: { 'Content-Type': 'application/json' },
+                                                  body: JSON.stringify(graphQLParams),
+                                                })
+                                                  .then(response => response.json())
+                                                  .catch(() => response.text());
+                                              ReactDOM.render(
+                                                React.createElement(GraphiQL, { fetcher: graphQLFetcher }),
+                                                document.getElementById('graphiql'),
+                                              );
+                                            </script>
+                                          </body>
+                                        </html>
+                                        """
+
+        return Vapor.Response(status: .ok, headers: ["Content-Type": "text/html"], body: html)
+    }
+
+    private func graphqlServer(_ req: Vapor.Request) throws -> EventLoopFuture<String> {
+        guard let body = req.body.string else {
+            throw NetworkError.noBody
+        }
+
+        // Create Swift dict
+        var query = "{}"
```

Comment:
I'm pretty sure you can leave this unintialized, since there's shouldn't be a case where we run a query without there being a query in the request